api/tests/Deck.test.js

```diff
@@ -0,0 +1,227 @@
+import Deck from '../services/deckService';
+
+let whiteCardsInPlay = [
```

Comment:
I would probably have all these in a tests/data folder that's then imported. but it's not very important so maybe better if you move onto a new ticket so we can get something working!